Cargo.toml

```diff
@@ -59,5 +59,5 @@ cargo_miri = ["cargo_metadata", "directories", "rustc_version"]
 rustc_tests = []
 
 [dev-dependencies]
-compiletest_rs = { version = "0.3.22", features = ["tmp", "stable"] }
+compiletest_rs = { version = "0.3.23", features = ["tmp"] }
```

Comment:
Did clippy make the same feature switch? If not, this will likely trigger the duplicate complication check when we update Miri in rustc.

Reply:
Yeah, half year ago: https://github.com/rust-lang/rust-clippy/commit/a8ce9d655ae600b707c72e31b2c18c4060e60ff4
:grinning:

Reply:
lol.^^
That makes me wonder why we did not trigger the aforementioned check all along. Oh well.

Reply:
It's under `dev-dependencies`, it doesn't end up in the binary.

Reply:
But the purpose of the check is ti reduce build times. Binary or not shouldn't matter.